test/integration/test-amp-analytics.js

```diff
@@ -17,11 +17,7 @@
 import {BrowserController, RequestBank} from '../../testing/test-helper';
 import {parseQueryString} from '../../src/url';
 
-// TODO(wg-analytics): These tests time out on Firefox and Safari
-// (locally too) which causes browser disconnections on SauceLabs.
-const t = describe.configure().skipSauceLabs();
-
-t.run('amp-analytics', function () {
+describe('amp-analytics', function () {
```

Comment:
Keep the TODO of "These tests time out on Firefox and Safari (locally too)"

Reply:
Another good catch. They do indeed time out, so I skipped them on FF / Safari.